packages/arb-token-bridge-ui/src/hooks/TransferPanel/useSetInputAmount.ts

```diff
@@ -17,5 +17,10 @@ export function useSetInputAmount() {
     [decimals, setQueryParams]
   )
 
-  return setAmount
+  const setAmount2 = useCallback(
+    (newAmount: string) => setQueryParams({ amount2: newAmount }),
+    [setQueryParams]
+  )
```

Comment:
i think this should be highly similar to the `setAmount` to handle decimals?

Reply:
updated